main/acle.md

```diff
@@ -1503,21 +1508,28 @@ instructions include AES{E, D}, SHA1{C, P, M} and others. This also implies
 
 #### AES extension
 
-`__ARM_FEATURE_AES` is defined to 1 if the AES Crypto instructions from
-Armv8-A are supported and intrinsics targeting them are available. These
-instructions include AES{E, D}, AESMC, AESIMC and others.
+`__ARM_FEATURE_AES` is defined to 1 if the AES Crypto instructions
+from Armv8-A are supported and intrinsics targeting them are
+available. These instructions are identified by `FEAT_AES` and
+`FEAT_PMULL` in [[ARMARMv8]](#ARMARMv8), and they include AES{E, D},
+AESMC, AESIMC and others.
 
 #### SHA2 extension
 
-`__ARM_FEATURE_SHA2` is defined to 1 if the SHA1 & SHA2 Crypto instructions
-from Armv8-A are supported and intrinsics targeting them are available. These
-instructions include SHA1{C, P, M} and others.
+`__ARM_FEATURE_SHA2` is defined to 1 if the SHA1 & SHA2 Crypto
```

Comment:
Would it make sense to replace "SHA1 & SHA2" with "SHA1 & SHA-256" or "SHA1 & SHA2-256"? The `FEAT_*` references fix the ambiguity but, as noted in the bug, I don't see where "SHA2 Crypto Instructions" is defined at all.

The Arm Architecture Reference Manual calls this the "SHA1 and SHA2-256 functionality", not the "SHA1 and SHA2 functionality". This would both match the reference manual *and* be clearer.

Reply:
good idea! ack